include/marl/scheduler.h

```diff
@@ -64,6 +69,8 @@ class Scheduler {
 
   // unbind() unbinds the scheduler currently bound to the current thread.
   // There must be a existing scheduler bound to the thread prior to calling.
+  // unbind() flushes any enqueued tasks on the single-threaded worker before
```

Comment:
What do you mean here by the "single-threaded worker" do you mean current worker thread?

Reply:
See https://github.com/google/marl/issues/79#issuecomment-584277414.
I'll try setting aside some time this week to write up some proper documentation here.